plenum/test/checkpoints/test_message_outside_watermark.py

```diff
@@ -50,13 +50,13 @@ def test_non_primary_recvs_3phase_message_outside_watermarks(
     slowNode.nodeIbStasher.delay(chk_delay(300))
 
     initialDomainLedgerSize = slowNode.domainLedger.size
-    oldStashCount = slowReplica.spylog.count(TestReplica.stashOutsideWatermarks.__name__)
```

Comment:
I think it's better to check for `slowReplica.stasher.num_stashed_watermarks`

Reply:
Done.